### DISCUSSION TITLE
Can we have a new minor release?

### DISCUSSION THREAD

**Discussion by andrewL-avlq** [eclipse/tm4e#527] [General]
Hi,
I would appreciate it if there could be a minor release prepared so that we can pick up the latest bug fixes.
Is now a good time for this?

**Comment by sebthom** [comment]
@mickaelistria wdyt? can you create a release or shall we wait for #502 (which I don't know when/if it is happening)?

**Reply by mickaelistria** [reply]
I think it's OK to release now if there is demand; and maybe re-release after #502 if there is another demand. I will add some notes about how to release in contributing.md and will then perform the release.

**Reply by mickaelistria** [reply]
I've created https://github.com/eclipse/tm4e/blob/main/CONTRIBUTING.md#release with the involved steps. @sebthom would you have the opportunity to try those and validate they are correct and nothing is missing?

**Comment by sebthom** [comment]
I just published release 0.7.0.

**Reply by mickaelistria** [reply]
Thanks a lot!

**Reply by andrewL-avlq** [reply]
Thanks!
